### PR TITLE
Update all the things

### PR DESCRIPTION
**Pull request by ttill**
Updated all dependencies including ember (using ember-cli-update).
In addition I ported the import of `easing` to `ember-auto-import`.

### CODE REVIEW

**Comment by jakebixbyavalara** [comment]
Looks good aside from the linting issues breaking the travis build. I don't have permissions on the repo anymore, but I'm sure once you fix the linting issues @psheraton would be happy to merge and cut a new major version. The upgrade to `ember-auto-import` would be great to get out.

**Comment by ttill** [comment]
Thanks for your feedback.
I fixed the remaining eslint issues. I am unable to figure out however why template-lint is linting inside node_modules. I've never seen this before.

**Comment by jakebixbyavalara** [comment]
@ttill I think the `/` you've added on some of the paths may be tripping things up. If you want to include directories, I think you want to use a wildcard like `/tmp/*` for instance. Looking at the `.eslintignore` docs they list that the `node_modules` directory is ignored by default (also lists it as `/node_modules/*`): https://eslint.org/docs/user-guide/configuring#eslintignore

**Comment by ttill** [comment]
I didn't actually manually modify this file.  This is what ember-cli-update did to it. It does no cause problems in other projects of mine.

Also see: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/.eslintignore

File: package.json [hunk 1]
```diff
@@ -6,72 +6,70 @@
...
   "author": "Jake Bixby",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "repository": "https://github.com/peopleconnectus/ember-concurrency-scroll.git",
   "scripts": {
     "build": "ember build",
-    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "lint:hbs": "ember-template-lint .",
```
Comment (psheraton) [inline package.json]: Can you either target the template files in this app/addon folders directly on this line or add `*` to the end of `/`s like @jakebixbyavalara recommended. Cause as it stands now the `lint:hbs` command is failing due to it incorrectly testing the node_module template files:

image

Comment (ttill) [reply package.json]: The line is as proposed by ember-cli and it works like this in other projects.

See: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json

**Comment by ttill** [comment]
Looks like our linting problems were caused by npm messing up its lock file during some update. After deletion + recreation everything works now.
I also updated everything again to latest versions (which by itself had no effect on the linting problems).

**Comment by psheraton** [comment]
Merged and Released as version v1.0.0

Thanks for your PR!

**Pull request merged**